README.md

```diff
@@ -50,17 +52,21 @@ Several template variables are made available to the script running inside the t
 * `title` subject (%s)
 * `messageLines` array of body lines (%b)
 
-`dateFnsFormat` is the date-fns [format](https://date-fns.org/docs/format) function. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for sample usage.
+`dateFnsFormat` is the date-fns [format](https://date-fns.org/docs/format) function. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for usage example.
+
+`options` the object documented below. Useful for parsing the repository name. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for sample usage.
+
+`request` is an instance of [sync-request](https://www.npmjs.com/package/sync-request). This can be useful for querying the Jira API for example to extract extra metadata about a ticket related to a commit.
```

Comment:
I don't think we are going to come to agreement architecturally on this. Next week when I have time I plan to hard fork this and publish my version to npm.
I trust you understand. cheers!